base/src/main/java/com/mindera/skeletoid/logs/utils/ShareLogFilesUtils.java

```diff
@@ -68,15 +68,7 @@ public static void sendLogs(Activity activity, String intentChooserTitle, String
 
         final Intent intent;
 
-        //TODO Currently this only supports 1 file. The code commented would support multiple.
```

Comment:
Would it make sense to uncomment that code?

Reply:
So, for now I actually removed the code since I hadn't had time to test if it works.
This can be reverted (brought back) in the future I'd say.